From 5595e4d7bf3dc987058aa311150f8eb1fb388796 Mon Sep 17 00:00:00 2001
From: Steve Wainstead <wainstead@gmail.com>
Date: Fri, 21 Jan 2011 19:48:22 +0000
Subject: [PATCH 100/223] changing AC_CONFIG_HEADER to AC_CONFIG_HEADERS per
 the No Starch Press book on autotools; claims the
 newer macro is more functional, though the
 'configure' script it output is no different.

---
 server/configure.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/server/configure.in b/server/configure.in
index ed84600..f535f0a 100644
--- a/server/configure.in
+++ b/server/configure.in
@@ -18,7 +18,7 @@ dnl
 AC_PREREQ([2.66])
 AC_INIT([Waverous LambdaMOO], [beta1], [wainstead@gmail.com], [waverous-lambdamoo], [http://code.google.com/p/waverous/])
 AC_CONFIG_SRCDIR([functions.c])
-AC_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 
 
 
-- 
1.7.9.5

